library/cwm/src/lib/cwm/multi_status_selector.rb

```diff
@@ -175,7 +175,17 @@ def find_item(needle)
 
     # Updates the content based on items list
     def refresh
-      content.value = items.map(&:to_richtext).join("<br>")
+      new_value = items.map do |item|
+        item_content = item.to_richtext
+
+        if Yast::UI.TextMode
+          "#{item_content}<br>"
```

Comment:
this will create <br> at the end. Do you try it in ncurses if it does not force empty line at the end? ( compared to previous code )

Reply:
Sure, I'll do

Reply:
and test case for it?

Reply:
It seems to produce an acceptable output, without a new line after the latest item.

See the screenshot below.

![Screenshot_20200313_044228](https://user-images.githubusercontent.com/1691872/76604464-aa530c80-6506-11ea-8ad2-666040e3b840.png)

Reply:
> and test case for it?

Done